container_opts_unix.go

```diff
@@ -199,8 +198,11 @@ func remapRootFS(mounts []mount.Mount, uid, gid uint32) error {
 			return err
 		}
 	}
-	defer unix.Unmount(root, 0)
-	return filepath.Walk(root, incrementFS(root, uid, gid))
+	err = filepath.Walk(root, incrementFS(root, uid, gid))
+	if uerr := mount.Unmount(root, 0); err == nil {
+		err = uerr
+	}
```

Comment:
In the case where both error do you think it would make sense to `log.Warn()` the unmount error? Or is the error message likely to be basically the same as the original walk error and doesn't provide any extra information?